theme/src/site/_fonts.scss

```diff
@@ -1,36 +1,43 @@
+// The fonts were generated using fontpie (https://github.com/pixel-point/fontpie),
+// a tool that converts font files into web-compatible formats and generates the necessary CSS code.
 
-//== Font loading
 
-@mixin fontface($name, $file, $weight: normal, $style: normal) {
-    @font-face {
-        font-weight: $weight;
-        font-family: '#{$name}';
-        font-style: $style;
-        src: url('resources/fonts/#{$file}.ttf') format('truetype');
-    }
-}
+@font-face {
+    font-family: 'defaultFont';
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url('./resources/fonts/SourceSansPro/SourceSansPro-Regular.ttf') format('truetype');
```

Comment:
I just noticed we're using ttf fonts. I think we can replace them with woff2 fonts which is modern and offers better performance since it's compressed.

Reply:
replaced with woff2